projects/storefrontlib/src/lib/product/components/product-list/product-facet-navigation/product-facet-navigation.component.html

```diff
@@ -22,19 +22,20 @@ <h4 class="cx-search-facet-filter__header">
           {{facet.name}}
         </a>
       </span>
-      <div id="facet1" [ngbCollapse]="isFacetCollapsed(facet.name)">
+      <div id="facet_{{facetId}}" [ngbCollapse]="isFacetCollapsed(facet.name)">
```

Comment:
I'm wondering if we need `id` attribute for whole facet block. I browsed all project files and there was only one occurence of `facet1` already in this file. In my opinion we can remove this attribute.

Reply:
yes, since we used `facet1` I can get rid of that - it doesn't look like something necessary tbh